clap_complete/tests/testsuite/dynamic.rs

```diff
@@ -589,6 +589,45 @@ val3
         ]
     );
 }
+fn suggest_custom_valuehint() {
```

Comment:
nit: let's give space

Also, this is missing `#[test]`

Reply:
Sorry, these are lost during rebase.